go.mod

```diff
@@ -1,10 +1,49 @@
+module github.com/nickvanw/ircx/v2
 
 go 1.10
 
 require (
+	4d63.com/gochecknoglobals v0.0.0-20180908201037-5090db600a84 // indirect
+	4d63.com/gochecknoinits v0.0.0-20180528051558-14d5915061e5 // indirect
+	github.com/BurntSushi/toml v0.3.1 // indirect
+	github.com/alecthomas/gocyclo v0.0.0-20150208221726-aa8f8b160214 // indirect
+	github.com/alecthomas/gometalinter v2.0.12+incompatible // indirect
+	github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf // indirect
+	github.com/alexflint/go-arg v1.0.0 // indirect
+	github.com/alexkohler/nakedret v0.0.0-20171106223215-c0e305a4f690 // indirect
+	github.com/client9/misspell v0.3.4 // indirect
+	github.com/fatih/gomodifytags v0.0.0-20180914191908-141225bf62b6 // indirect
 	github.com/go-kit/kit v0.8.0
-	github.com/go-logfmt/logfmt v0.4.0 // indirect
-	github.com/go-stack/stack v1.8.0 // indirect
+	github.com/golang/lint v0.0.0-20181217174547-8f45f776aaf1 // indirect
+	github.com/google/shlex v0.0.0-20181106134648-c34317bd91bf // indirect
+	github.com/gordonklaus/ineffassign v0.0.0-20180909121442-1003c8bd00dc // indirect
+	github.com/jgautheron/goconst v0.0.0-20170703170152-9740945f5dcb // indirect
+	github.com/kisielk/errcheck v1.2.0 // indirect
+	github.com/mdempsky/maligned v0.0.0-20180708014732-6e39bd26a8c8 // indirect
+	github.com/mdempsky/unconvert v0.0.0-20180703203632-1a9a0a0a3594 // indirect
+	github.com/mibk/dupl v1.0.0 // indirect
+	github.com/nicksnyder/go-i18n v1.10.0 // indirect
+	github.com/nickvanw/ircx v0.0.0-20190104230957-b8ba1369554e
+	github.com/nsf/gocode v0.0.0-20181120081338-6cac7c69a41e // indirect
+	github.com/opennota/check v0.0.0-20180911053232-0c771f5545ff // indirect
+	github.com/pelletier/go-toml v1.2.0 // indirect
+	github.com/rogpeppe/godef v1.1.1 // indirect
+	github.com/securego/gosec v0.0.0-20181211171558-12400f9a1ca7 // indirect
+	github.com/sqs/goreturns v0.0.0-20181028201513-538ac6014518 // indirect
+	github.com/stripe/safesql v0.0.0-20171221195208-cddf355596fe // indirect
+	github.com/tpng/gopkgs v0.0.0-20180428091733-81e90e22e204 // indirect
+	github.com/tsenart/deadcode v0.0.0-20160724212837-210d2dc333e9 // indirect
+	github.com/walle/lll v0.0.0-20160702150637-8b13b3fbf731 // indirect
+	github.com/zmb3/goaddimport v0.0.0-20170810013102-4ab94a07ab86 // indirect
+	github.com/zmb3/gogetdoc v0.0.0-20190101160103-a2964ac3d615 // indirect
+	golang.org/x/lint v0.0.0-20181217174547-8f45f776aaf1 // indirect
+	golang.org/x/tools v0.0.0-20190104182027-498d95493402 // indirect
+	gopkg.in/alecthomas/kingpin.v3-unstable v3.0.0-20180810215634-df19058c872c // indirect
 	gopkg.in/sorcix/irc.v1 v1.1.4
+	gopkg.in/sorcix/irc.v2 v2.0.0-20180626144439-63eed78b082d
+	gopkg.in/yaml.v2 v2.2.2 // indirect
+	honnef.co/go/tools v0.0.0-20190104064624-c51f1e51d5a3 // indirect
+	mvdan.cc/interfacer v0.0.0-20180901003855-c20040233aed // indirect
+	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
+	mvdan.cc/unparam v0.0.0-20181201214637-68701730a1d7 // indirect
```

Comment:
Wow, where are all these extra indirect deps coming from? Did you do a `go mod tidy`? You're only bumping sorcix/irc, so I doubt it's its fault.

Reply:
`go mod tidy` was indeed not run, 908feb7 looks a lot better 😄